Configure.ps1

```diff
@@ -82,74 +81,75 @@ of the visual studio solution (App.Config and Web.Config) so that they are consi
     # Active Directory Tenant. This is a GUID which represents the "Directory ID" of the AzureAD tenant into which you want to create the apps.
     # Look it up in the Azure portal in the "Properties" of the Azure AD. 
     # IN THE FUTURE: We'd like to be able to get the tenant name from the tenant ID (or pass the tenant name as an argument)
-    $tenantName = $tenantId
+
+    # Login to Azure PowerShell (interactive: you'll need to sign-in with creds enabling your to create apps in the tenant)
+    $creds = Connect-AzureAD -TenantId $tenantId
+     if (!$tenantId)
```

Comment:
Is there a reason why this `if` block is indented by 1 character from the surrounding lines?